crates/language-tests/tests/language/statements/define/table/type_normal.surql

```diff
@@ -9,7 +9,7 @@ value = "[{ id: thing:2gmswqs3pk5j8a5s5tql }]"
 skip-record-id-key = true
 
 [[test.results]]
-match = '''string::matches($error,"Found record: `thing:.*` which is not a relation, but expected a  NORMAL")'''
+match = '''string::matches($error, /Found record: `thing:.*` which is not a relation, but expected a  NORMAL/)'''
```

Comment:
Does using string as a regex no longer work here? Or is this changed for a different reason.

Reply:
There is another test which still uses a string as input, I just modified one of them to show that string and regex inputs both work. There was no reason to change this otherwise.